tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -6,6 +6,5 @@ OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-3.0}
 LIBC="gnu"
 PACKAGES="kata-packages-uvm"
-[ "$CONF_GUEST" = yes ] && PACKAGES+=" kata-packages-uvm-coco"
```

Comment:
I don't think we should remove this. Let's keep it - only for now, the set of packages would be common.

Reply:
Got it. Added back